app/src/main/res/layout/main_persistent_bottom_sheet.xml

```diff
@@ -98,12 +98,12 @@
             android:layout_height="wrap_content"
             style="@style/h4"
             android:textColor="@color/white"
-            android:text="10000"/>
+            tools:text="10000"/>
 
         <TextView
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="Positive"
+            android:text="Death"
```

Comment:
Better to move this into a string resource file. You can do this by right clicking on it and choosing extract string resource.